### ISSUE TITLE
Add "list of upvars" option

### ISSUE THREAD

**Issue by rdococ** [jmoenig/Snap#2216]
The "list of upvars" would act similarly to the input in the "script variables" block - the "uplist" as I would call it would be accessed as a list within the custom block, which the block could add items to before returning. The caller would then be able to access each item of the uplist individually as separate upvars.

**Comment by brianharvey** [comment]
I don't see it.  The user has to _write_ the script containing these upvars _before_  running it, i.e., before the custom block has a chance to add upvars.  So, how in practice can the user take advantage of them?

**Comment by brianharvey** [comment]
No, for that you'd want to alternate upvars and values, I think, so you'd want the grouping-inputs feature, with a group of upvar-and-any.  But I may be wrong -- it's late at night and was in New York earlier today, so I'm zonked.
